sotodlib/site_pipeline/make_nearby_sources_db.py

```diff
@@ -0,0 +1,254 @@
+import os
```

Comment:
Let's call this script "update_sources_db.py".  "update" is used (ideally) for things that need to run, periodically, to add more info for the latest obs.

This new site_pipeline script needs an entry in docs/site_pipeline.rst.  You can document the module here, and then pull it in automatically (as is done for update_obsdb.py, e.g.), or put all the detail directly in site_pipeline.rst (as is done for, e.g. make_wafer_info.py).  At a minimum it should have an example config file.